### PR TITLE
Change user profile major to foreign key

### PR DESCRIPTION
**Pull request by churichard**
This is so that the production website's database will migrate properly

Also fixed a minor annoyance where the text in a course card sometimes moved to the next line on hover (because of the appearance of course info/eval buttons on the right)

### CODE REVIEW

File: tigerpath/models.py [hunk 1]
```diff
@@ -3,9 +3,9 @@
 from django.dispatch import receiver
 from django.conf import settings
 from django.contrib.auth.models import User
-from django.contrib.postgres.fields import JSONField
+from django.contrib.postgres.fields import JSONField, ArrayField
+
```
Comment (bnehoran) [inline tigerpath/models.py:7]: Why the extra line?

File: tigerpath/models.py [hunk 2]
```diff
@@ -155,7 +155,7 @@ class UserProfile(models.Model):
                                 on_delete=models.CASCADE,
                                 related_name='profile')
     nickname = models.CharField(max_length=50, null=True, blank=True)
-    major = models.CharField(max_length=7, null=True)
+    major = models.ForeignKey(Major, related_name="+", on_delete=models.CASCADE, null=True)
```
Comment (danielleung27) [inline tigerpath/models.py:158]: Why is related_name="+"

**Review by danielleung27 — APPROVED** [review]

File: tigerpath/models.py (hunk 2 above)
Comment (churichard) [reply tigerpath/models.py:158]: This means that majors don't link back to user profiles

File: tigerpath/models.py (hunk 1 above)
Comment (churichard) [reply tigerpath/models.py:7]: It's convention to put an extra line between "from" imports and standard imports

**Pull request merged**